packages/engine/src/worker/runner/python/mod.rs

```diff
@@ -256,3 +257,36 @@ async fn _run(
     // }
     Ok(())
 }
+
+pub(crate) fn cleanup_python_runner(experiment_id: &ExperimentId, exit_status: EngineExitStatus) {
+    // Cleanup python socket files in case the engine didn't
+    let frompy_files = glob::glob(&format!("{experiment_id}-frompy*"));
+    let topy_files = glob::glob(&format!("{experiment_id}-topy*"));
+
+    // We're ignoring glob errors as they shouldn't stop the whole cleanup process.
```

Comment:
See my comments above about glob errors. I also want to be clear that we have a `Result` from `glob::glob` and we have a collection of `Result`s from `into_iter`. The first one stops the rest of it.

Reply:
Changed in https://github.com/hashintel/hash/pull/494/commits/bab18b3df1a73a1cde8667aa505957f80096cf90